### ISSUE TITLE
Updates to screen colors for section 508 compliance and inclusion in User Guide

### ISSUE THREAD

**Issue by lbross** [USFS-PNW/Fia-Biosum-Manager#303]
The BioSum color scheme seems to be causing print-outs to look terrible. I do expect that some folks will want to print out the users guide and we would like to make the outcome of such printing less disappointing. The gray desk top and the gray buttons on gray background  (in the table of contents section on the left side of the desktop) seem to be the crux of the problem. The contrast is inadequate to print well AND the contrast on screen for the buttons is probably too low to be Section 508 compliant. Moreover, when printed on a laser printer, these gray areas come out with some kind of interference pattern or stripes. Can these colors (desktop, buttons and button text) be customized in Settings without too much difficulty? I’d like to set some of them to white (and the text to black, or perhaps to white if we set buttons to black or blue) before printing.

CAP (Comms and applications program) told us to put the whole users guide in one word file and to simplify its format. Sebastian will work on that as he has time. We aim to complete the Users Guide by early Fall. It will take 9 months to get published, even electronically, as layout takes quite a while as practiced by CAP.
